plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteConnectorTest.java

```diff
@@ -95,6 +95,35 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
         }
     }
 
+    /**
+     * Test that we are correctly escaped the wildcard characters while requesting DataBaseMetaData: getSchemas and getTables with params contains pattern.
+     */
+    @Test
+    public void testDatabaseMetadataSearchEscapedWildCardCharacters()
+    {
+        // wildcard characters on schema name
+        assertQuerySucceeds("SHOW TABLES FROM public");
+        assertQueryFails("SHOW TABLES FROM \"publi_\"", ".*Schema 'publi_' does not exist");
+        assertQueryFails("SHOW TABLES FROM \"pu%lic\"", ".*Schema 'pu%lic' does not exist");
+
+        try {
+            assertUpdate("CREATE TABLE abcd (a int, b int, c int) WITH (primary_key = ARRAY['a'])");
+            assertUpdate("CREATE TABLE ab_d (a int, b int, c int) WITH (primary_key = ARRAY['a'])");
+            assertUpdate("CREATE TABLE \"a%d\" (a int, b int, c int) WITH (primary_key = ARRAY['a'])");
```

Comment:
Specify different `primary_key` in each table definition like below and assert the table definition. 
* abcd → a
* ab_d → b
* \"a%d\" → c